### PR TITLE
fix(typo): lingual corrections and correction of tool names

### PR DESCRIPTION
**Pull request by Sleinmaster**
## Summary

- corrected punctuation and several typos
- consistent use of the official capitalization for GitHub and some other tools/brands on the same pages

### Location

https://docs.flybywiresim.com/dev-corner/dev-guide/resources/
https://docs.flybywiresim.com/dev-corner/dev-guide/setup-environment/
https://docs.flybywiresim.com/dev-corner/dev-guide/specific/flypad-dev/
https://docs.flybywiresim.com/dev-corner/development-projects/documentation/
https://docs.flybywiresim.com/dev-corner/development-projects/website/
https://docs.flybywiresim.com/fbw-a32nx/a32nx-api/a32nx-flightdeck-api/
https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/
https://docs.flybywiresim.com/fbw-a32nx/support/exp/
https://docs.flybywiresim.com/fbw-a32nx/support/
https://docs.flybywiresim.com/simbridge/troubleshooting/

Discord username (if different from GitHub):Slein#8982

### CODE REVIEW

**Comment by Sleinmaster** [comment]
Leaving it at that for now and might open a new PR in the future where I check the pages in a more structured way.

File: docs/dev-corner/dev-guide/resources.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 ## Support from the FlyByWire Team on Discord
 
-To get additional information and support please make sure you join our Discord. There are various channels dedicated to support developers (in fact most channels).
+To get additional information and support, please make sure you join our Discord. There are various channels dedicated to support developers (in fact most channels).
 
 [:fontawesome-brands-discord:{: .discord } - **Discord Link**](https://discord.gg/flybywire){target=new}
 
 First get yourself the "programmer" role in [#roles](https://discord.com/channels/738864299392630914/751780817772216401/816730253543604224){target=new}.
 
 The most  general channel for getting help for development is [#dev-support](https://discord.gg/v3jAxJpwZm){target=new}.
```
Comment (Valastiri) [inline docs/dev-corner/dev-guide/resources.md]:
```suggestion
The most general channel for getting help for development is [#dev-support](https://discord.gg/v3jAxJpwZm){target=new}.
```

**Review by Valastiri — APPROVED** [review]
Minor change I noticed in the middle of double checking. Otherwise LGTM

File: docs/dev-corner/dev-guide/resources.md (hunk 1 above)
Comment (Sleinmaster) [reply docs/dev-corner/dev-guide/resources.md]: In this context I think "most" is the correct word, but this two versions would be possible:

- The most  general channel for getting help for development is [#dev-support]
- The best channel for getting help for development is [#dev-support]

Comment (Valastiri) [reply docs/dev-corner/dev-guide/resources.md]: Ah that's a fair point -- although the double space still exists i'll fix my review and if you want to commit via review that would work :)

**Comment by Valastiri** [comment]
lgtm thanks again Slein!

**Pull request merged**